generators/app/package-json.js

```diff
@@ -0,0 +1,268 @@
+'use strict';
+
+var _ = require('lodash');
+var util = require('util');
+
+var gruntify = function(scripts) {
+  var result = {};
+  scripts.forEach(function(script) {
+    result[script] = 'grunt ' + script;
+  });
+  return result;
+};
+
+/**
+ * Replaces each "%s" in the string with the package name from the context.
+ *
+ * @example
+ *   nameify('foo %s bar %s', {packageName: 'baz'}) === 'foo baz bar baz'
+ * @param  {String} string
+ * @param  {Object} context
+ * @return {String}
+ */
+var nameify = function(string, context) {
+  var args = [string];
+  string.match(/%s/g).forEach(function() {
+    args.push(context.packageName);
+  });
+  return util.format.apply(util, args);
+};
+
+var KARMA_BROWSERS = ['chrome', 'firefox', 'ie', 'opera', 'safari'];
+
+var PACKAGE = {
+
+  /**
+   * Common package.json properties.
+   *
+   * @function common
+   * @param    {Object} context
+   *           Rendering context passed in by the generator.
+   * @return   {Object}
+   */
+  common: function(context) {
+    var pkg = {
+      name: context.packageName,
+      author: context.author,
+      license: context.licenseName,
+      version: '0.0.0',
+      main: 'src/plugin.js',
+      keywords: [
+        'videojs',
+        'videojs-plugin'
+      ],
+      'browserify-shim': {
+        'qunit': 'global:QUnit',
+        'sinon': 'global:sinon',
+        'video.js': 'global:videojs'
+      },
+      standard: {
+        ignore: [
+          '**/dist/',
+          'docs'
+        ]
+      },
+      scripts: {
+        'build': '',
+        'build:css': '',
+        'build:js': '',
+        'build:test': '',
+        'clean': '',
+        'docs': 'documentation src/*.js -f html -o docs/api',
+        'lint': 'standard .',
+        'start': '',
+        'test': '',
+        'preversion': './scripts/npm-preversion.sh',
+        'version': './scripts/npm-version.sh',
+        'postversion': './scripts/npm-postversion.sh',
+        'watch': '',
+        'watch:css': '',
+        'watch:js': '',
+        'watch:test': ''
+      },
+      dependencies: {},
+      devDependencies: {
+        'babel': '^5.8.0',
+        'babelify': '^6.0.0',
+        'browserify': '^11.0.0',
+        'browserify-shim': '^3.0.0',
+        'documentation': '^3.0.0',
+        'global': '^4.3.0',
+        'karma': '^0.13.0',
+        'karma-browserify': '^4.4.0',
+        'karma-chrome-launcher': '^0.2.0',
+        'karma-detect-browsers': '^2.0.0',
+        'karma-firefox-launcher': '^0.1.0',
+        'karma-ie-launcher': '^0.2.0',
+        'karma-opera-launcher': '^0.3.0',
+        'karma-qunit': '^0.1.0',
+        'karma-safari-launcher': '^0.1.0',
+        'npm-run-all': '^1.2.0',
+        'qunitjs': '^1.0.0',
+        'sinon': '^1.0.0',
+        'video.js': '^5.0.0',
```

Comment:
if we're doing the split build as in https://github.com/brightcove/videojs-dock/pull/3
`video.js` would need to be in `dependencies` instead.